frontend/src/pages/mainPage.js

```diff
@@ -76,16 +76,24 @@ function MainPage() {
         try {
             setLoading(true);
             const response = await axios.post(
-                `${apiEndpoints.url}${apiEndpoints.games.getGame}`,
+                `${apiEndpoints.url}${apiEndpoints.games.getGames}`,
                 {
-                    page: page + 1,
-                    per_page: gamesPerSlide
+                    pagination: {
+                        page: page + 1,
+                        per_page: gamesPerSlide,
+                    },
+                    filters: [
+                        {
+                            'filter_name': 'sport_id'
```

Comment:
Magic strings...
We should have our filters specified somewhere in one place and then just reuse them

Could you do that?